src/core/repository/account.py

```
@@ -46,81 +58,179 @@ def __init__(self, session: Session):
-        query_subscription_insert = insert(subscription_table).values(
-            account_id=id_, type=type_
+        (account_id, account_created_at) = deepcopy(account_row)
+        query_subscription_insert = (
+            insert(subscription_table)
+            .values(account_id=account_id, type=type_)
+            .returning(subscription_table.c.id, subscription_table.c.created_at)
         )
-        self.session.execute(query_subscription_insert)
+        subscription_row = self.session.execute(query_subscription_insert)
+        if subscription_row is None:
+            raise SystemError("Something on database did not return")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)